packages/yum_repositories/saltstack.repo

```diff
@@ -1,6 +1,6 @@
 [saltstack]
 name=SaltStack repo for RHEL/CentOS $releasever
-baseurl=https://repo.saltstack.com/yum/redhat/$releasever/$basearch/archive/2018.3.3
+baseurl=https://repo.saltstack.com/yum/redhat/$releasever/$basearch/archive/2018.3.4
```

Comment:
Out of the scope of this PR, but shouldn't we "templatize" all those use of the Salt version (using `SALT_MASTER_IMAGE_SALT_VERSION` which is defined in our buildchain)?

Ideally, bumping the Salt version would require to edit only one of two place and not many files like we currently have to do (it's error prone).

Reply:
#640 

I think the 'best' approach, especially given with `doit` we can use whichever input (format) we want (which is less the case with `make`) is to have a YAML file in the `salt` tree which can be imported in SLS files to retrieve all versions, and can also be read by the `doit` modules to do whatever they want. We have more flexibility in the `doit` modules than in the Salt files, hence embedding in Salt and consuming from `doit` rather than embedding in `doit` and somehow embedding in Salt, unless we'd create a Salt-readable file from the `doit` data in some step, but that seems like a useless indirection to me.

Reply:
> #640

Ha we already have an issue for that, perfect!

> I think the 'best' approach, especially given with `doit` we can use whichever input (format) we want (which is less the case with `make`) is to have a YAML file in the `salt` tree which can be imported in SLS files to retrieve all versions, and can also be read by the `doit` modules to do whatever they want.

We would need to add a YAML parser to the build virtualenv, but that's not a big dependency (and it's better than generating Salt-readable file, as you mentioned, which would probably require a YAML lib as well anyway).

Reply:
You may want to discuss this in the #640 instead ;-)